raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -1876,12 +1836,12 @@ private void appendBatch(
     }
 
     private long maybeAppendBatches(
-        LeaderState state,
+        LeaderState<T> state,
         long currentTimeMs
     ) {
-        long timeUnitFlush = accumulator.timeUntilDrain(currentTimeMs);
+        long timeUnitFlush = state.accumulator().timeUntilDrain(currentTimeMs);
```

Comment:
While we're here, can we fix the name? It should be `timeUntilFlush`.

Reply:
The name of the method or the variable?

Reply:
@hachikuji we had this discussion in the PR that introduced the `BatchAccumulator`. I was under the impression that we agreed to not use the word "flush" since in Java that word is usually used to represent some kind of IO sync.

Reply:
There is an IO sync happening with the LeaderChangeMessage. That is the only time accumulator.flush is called. I think maybe this variable name can get changed to timeUntilDrain?

Reply:
I was just interested in the spelling fix 😉 . I am fine with `timeUntilDrain` as well.